saas_demo/doc/index.rst

```diff
@@ -39,6 +39,17 @@ Following commands deploy the system on 80 port. You may need to change that to
    sed -i -e "s/automatic_addons_path_update = fields\.Boolean(default=True)/automatic_addons_path_update = fields.Boolean(default=False)/g" vendor/it-projects-llc/saas-addons/saas_demo/models/saas_operator.py
    sed -i "s/tools\.config\['data_dir'\], 'repos'/os\.path\.expanduser('~'), 'vendor'/g" vendor/it-projects-llc/saas-addons/saas_demo/os.py
 
+   # Set up the operator
+   YOUR_DOMAIN="apps.example.com"
+   YOUR_SCHEME="http"
+   # YOUR_SCHEME="https"
+
+   # Change if necessary
+   BUILD_URL_TEMPLATE="'http:\/\/' + obj()\.env\.cr\.dbname + '\.127\.0\.0\.1\.nip\.io:8069'"
+
+   sed -i "s/http:\/\/{db_name}.{db_id}.127.0.0.1.nip.io/$YOUR_SCHEME:\/\/{db_name}\.$YOUR_DOMAIN/g" vendor/it-projects-llc/saas-addons/saas/data/saas_operator_data.xml
```

Comment:
you can replace ``/`` with ``;`` to don't escape slash in strings:

 ``sed -i "s;WHAT;TO_WHAT;g"